src/components/Chart/ChartOptions.jsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React, { useEffect, useState } from 'react';
+import Dropdown from 'react-bootstrap/Dropdown';
+import Form from 'react-bootstrap/Form';
+import SelectableContext from 'react-bootstrap/SelectableContext';
+import { useDispatch, useSelector } from 'react-redux';
+import { NumberInlineInput } from 'pc-nrfconnect-shared';
+import { func, number } from 'prop-types';
+
+import { chartState, setYMax, setYMin } from '../../reducers/chartReducer';
+
+const unitLabels = ['\u00B5A', 'mA', 'A'];
+
+const ChartOptions = () => {
+    const { yMin, yMax, yAxisLock } = useSelector(chartState);
+    const dispatch = useDispatch();
+
+    const [localYMin, setLocalYMin] = useState(1);
+    const [localYMax, setLocalYMax] = useState(1);
+
+    /** Mapping to the power to get correct value of uA
+     * 0=uA, 1=mA, 2=A
+     */
+    const [unitPowerYMin, setUnitPowerYMin] = useState(0);
+    const [unitPowerYMax, setUnitPowerYMax] = useState(0);
+
+    /** Get the unit power
+     * @param {number} uA number av microAmpere
+     * @returns {0 | 1 | 2} the power in which it is preferred to scale the y value
+     */
+    const getUnitPower = uA => {
+        const absoluteValue = Math.abs(uA);
+        if (absoluteValue >= 1e6) {
+            return 2;
+        }
+        if (absoluteValue > 1000) {
+            return 1;
+        }
+        return 0;
+    };
```

Comment:
Is the purpose of this function understandable?

Reply:
Nice that you added JSDoc. I'm bad at remembering what numbers in this type of context mean, so to me it would be easier to remember if these were constants, like suggested above. Otherwise it looks good!